scripts/issues-from-release-notes.py

```diff
@@ -0,0 +1,65 @@
+import argparse
```

Comment:
If you add a line like

```
#!/usr/bin/env python3
```

(a "shebang"), then run

```
chmod +x scripts/issues-from-release-notes.py
```

then users can directly run scripts/issues-from-release-notes.py without needing to think about which Python version to use.

Reply:
Done. Thanks!